net/eoip/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2006-2012 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=eoip
+PKG_VERSION:=0.5
```

Comment:
If you later set the download for version 0.6.1, you should set also PKG_VERSION here to 0.6.1

Or, if you are piggybacking with an upstream source archive of version 0.5, you should still call this as version 0.5 instead of calling this as "0.6.1" and then providing a tar.gz of version 0.5 copied from upper upstream...

That looks strange in any case. It pretty much looks like your download site & "release 0.6.1" is just a storage of an old tar.gz downloaded from upstream. (instead of you really hosting the underlying source code as plain source).

Reply:
> One more comment: there needs to be just one commit. (Currently you have 5 commits that modify only 4 files...)

This is the first time I come across pull request , and by package openwrt requirements, I may close this one and create a new one, if it need